opty/direct_collocation.py

```diff
@@ -166,6 +172,15 @@ def __init__(self, equations_of_motion, state_symbols,
             ndarrays of floats of shape(N,). Any time varying parameters in
             the equations of motion not provided in this dictionary will
             become free trajectories optimization variables.
+        known_experiment_map : dictionary, optional
+            A dictionary that maps the state SymPy functions of time to
+            ndarrays of floats of shape(N,4). Any time varying parameters in
+            the equations of motion not provided in this dictionary will
+            become free trajectories optimization variables.
+	Lambda: float
```

Comment:
Yes, Lambda is a optional float number. If Lambda = 0, the homotopy will not applied. Yes, we could create a new Collocator subclass, but as I am new in python, I am not sure whether I could correctly change out all the relevant methods for the homotopy functionality.